frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -538,20 +554,40 @@ const CreateBackingStoreForm: React.FC<CreateBackingStoreFormProps> = withHandle
       },
       spec: {
         type: typeNoobaaMap[provider],
+        ssl: false,
+      },
+    };
+    if (provider === 'PVC') {
+      // eslint-disable-next-line
+      bsPayload.spec['pvPool'] = {
```

Comment:
why not `bsPayload.spec.pvPool`? Then you don't need disable eslint for next line.

Reply:
it dont work like that. Have a look  at spec object.

Reply:
ack